ui/src/translations/de.ts

```diff
@@ -74,6 +74,7 @@ export const de = {
     number_of_users: '{{count}} Benutzer',
     number_of_subscribers: '{{count}} Abonnenten',
     number_of_points: '{{count}} Punkte',
+    total_number_of_points: '{{count}} Gesamtpunkte',
```

Comment:
Its not necessary to add another field here, number of points is already good.

Reply:
I wanted to make it clear that this metric was the total point count and not points for another type of content.

Do I still need to fix this not that you've merged it?

Reply:
No I got it, I changed some other things around too.